GitCommands/Settings/AppSettings.cs

```diff
@@ -894,6 +894,12 @@ public static bool ShowStashCount
             set => SetBool("showstashcount", value);
         }
 
+        public static bool ShowAheadBehindData
+        {
+            get => GetBool("showAheadBehindData", true);
```

Comment:
most keys are lower case only

Reply:
We have a good deal of PascalCased settings as well, but this would be the first camelCased one. 
Agree with @gerhardol, let's align to either be all in a lower case or in a PascalCase